src/init.cpp

```diff
@@ -430,7 +430,7 @@ void SetupServerArgs(ArgsManager& argsman)
     argsman.AddArg("-prune=<n>", strprintf("Reduce storage requirements by enabling pruning (deleting) of old blocks. This allows the pruneblockchain RPC to be called to delete specific blocks and enables automatic pruning of old blocks if a target size in MiB is provided. This mode is incompatible with -txindex. "
             "Warning: Reverting this setting requires re-downloading the entire blockchain. "
             "(default: 0 = disable pruning blocks, 1 = allow manual pruning via RPC, >=%u = automatically prune block files to stay under the specified target size in MiB)", MIN_DISK_SPACE_FOR_BLOCK_FILES / 1024 / 1024), ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
-    argsman.AddArg("-reindex", "Rebuild chain state and block index from the blk*.dat files on disk. This will also rebuild active optional indexes.", ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
+    argsman.AddArg("-reindex", "Rebuild chain state and block index from the blk*.dat files on disk. This will also rebuild active optional indexes (only useable from command line, not configuration file)", ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
```

Comment:
You removed the period but I realize the punctuation is not consistent across the commands

Reply:
yeah, it's mixed in the file, but im used to (and prefer) not having punctuation in stdout help text

Reply:
you also mixed it here 😄
https://github.com/bitcoin/bitcoin/blob/79e5606e750b28fa2123f96ff07757c931ae5f91/src/util/system.cpp#L948-L955

Reply:
oops! good catch, ill make them consistent